### PR TITLE
Add test case for tf.reduce_sum(..., axis = ...) (issue #822)

### PR DESCRIPTION
**Pull request by mishun**

### CODE REVIEW

**Comment by mishun** [comment]
I noticed that CI build hasn't failed after merge. It doesn't run tests?

**Comment by Oceania2018** [comment]
It shows successful, is it supposed to failed?

**Comment by mishun** [comment]
Yes, new test case currently fails with NotImplementedException. From what I can see, CI build doesn't run test from `TensorFlowNET.Graph.UnitTest`.
